setup.sh

```diff
@@ -19,25 +19,28 @@ function getLatestK8s {
    echo $latestVersion
 }
 
-# Environment variables / positional parameters and defaults. 
+# Environment variables / positional parameters and defaults.
+
+# Add ~/bin to PATH for the script only 
+export PATH=$PATH:~/bin
 
 # TODO make $1 and $2 required?
-keyName=$1
+read -p "Enter the key name used to sign and verify: "  keyName
```

Comment:
I'd like to make these required so the user can't bypass it. What about something like: 

``` bash
while read -p "Enter the key name used to sign and verify. I.E. contoso-io: " keyName && [[ -z "$keyName" ]] ; do
    echo ''
    echo "A keyname is required."
    echo ''
done
```